codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/endpoint/generators/EndpointParamsInterceptorGenerator.kt

```diff
@@ -84,10 +84,10 @@ class EndpointParamsInterceptorGenerator(
             let input = context.input()?;
             let _input = input
                 .downcast_ref::<${operationInput.name}>()
-                .ok_or_else(|| "failed to downcast to ${operationInput.name}")?;
+                .ok_or("failed to downcast to ${operationInput.name}")?;
```

Comment:
Thanks for updating these. For some reason, I misremembered and thought `.ok_or` would be yelled at by `clippy`.